src/pyfaf/queries.py

```diff
@@ -808,10 +810,44 @@ def get_report_by_hash(db, report_hash):
     or None if not found.
     """
 
-    return (db.session.query(Report)
-                      .join(ReportHash)
-                      .filter(ReportHash.hash == report_hash)
-                      .first())
+    return find_report(db, report_hash)
```

Comment:
This is now redundant. Please rename find_report to get_report and drop get_report_by_hash renaming other occurrences as well.